types/v56/types.go

```diff
@@ -1134,6 +1134,14 @@ type PublishExternalCatalogParams struct {
 	PreserveIdentityInfoFlag *bool  `xml:"PreserveIdentityInfoFlag,omitempty"` // True includes BIOS UUIDs and MAC addresses in the downloaded OVF package. If false, those information will be excluded.
 }
 
+// PublishCatalogParams represents the configuration parameters of a catalog published to other orgs
```

Comment:
Nit, separator needed for two sentences:

```suggestion
// PublishCatalogParams represents the configuration parameters of a catalog published to other orgs.
```